Dockerfile

```diff
@@ -1,12 +1,27 @@
-FROM adoptopenjdk:11-jre-hotspot
-
-RUN apt-get update \
-    && apt-get install -y unzip \
-    && rm -rf /var/lib/apt/lists/*
+ARG VERSION=3.6
+ARG JAVA_VERSION=11-jre-hotspot
 
+FROM adoptopenjdk:${JAVA_VERSION}
 ARG VERSION
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+LABEL "maintainer"="DITA Open Toolkit Development Team"
```

Comment:
There is no _DTA Open Toolkit Development Team_, so I'm not sure what this should be.

Reply:
Typically this would be the legal entity that owns the copyright as found in the LICENCE file - The DITA-OT LICENSE hasn't listed the owner explicitly, so I would assume that the contributors to the repository - any suggestion as to how to rephrase it?

Reply:
In the docs, we use _“DITA Open Toolkit project”_ when we need a collective noun for the team. 

For [example](https://www.dita-ot.org/dev/topics/plugin-coding-conventions.html):

> … the DITA Open Toolkit project recommends …

Reply:
_DITA Open Toolkit project_ is good, since the maintainers really are just the people who file pull requests and people can come and go.

Reply:
_DITA Open Toolkit project_ it is then. 50b6181